### ISSUE TITLE
Night Ligts is applied over video

### ISSUE THREAD

**Issue by Kringlan** [elementary/wingpanel-indicator-nightlight#41]
Night light is applied over kodi on my secondary screen, a tv. I want to view video without distortion.

An option to apply Night light on a per monitor basis or to be disabled for certain applications would be much appreciated.

**Comment by thomorl** [comment]
I agree, but should you choose to disable night light for certain applications, it would be much appreciated if this behaviour could be configurable (via a list of apps to disable for; or a switch in Settings).
